go/cmd/dolt/commands/dump.go

```diff
@@ -94,6 +96,7 @@ func (cmd DumpCmd) ArgParser() *argparser.ArgParser {
 	ap.SupportsFlag(batchFlag, "", "Return batch insert statements wherever possible, enabled by default.")
 	ap.SupportsFlag(noBatchFlag, "", "Always emit one row per statement, rather than batching multiple rows into each statement.")
 	ap.SupportsFlag(noAutocommitFlag, "na", "Turns off autocommit for each dumped table. Used to speed up loading of outputted sql file")
+	ap.SupportsFlag(schemaOnlyFlag, "", "Dumping a table schema with no data to sql file")
```

Comment:
I was looking at our argument doc strings for the CLI the other day and we are mostly consistent around an imperative verb form to start them off (e.g. "Define the type",  "Return batch"). It would be good to try and standardize around that more so the docs look more consistent. I also added consistent punctuation at the end of the last two doc strings to match the others. 

```suggestion
	ap.SupportsFlag(noBatchFlag, "", "Emit one row per statement, instead of batching multiple rows into each statement.")
	ap.SupportsFlag(noAutocommitFlag, "na", "Turn off autocommit for each dumped table. Useful for speeding up loading of output SQL file.")
	ap.SupportsFlag(schemaOnlyFlag, "", "Dump a table's schema, without including any data, to the output SQL file.")
```